frontend/src/rules/general_rules/GeneralRuleEdit.tsx

```diff
@@ -15,127 +34,161 @@ const CustomToolbar = () => {
         </Toolbar>
     );
 };
+const GeneralRuleEdit = () => {
+    const transform = (data: any) => {
+        if (data.description == null) {
+            data.description = "";
+        }
+        if (data.scanner_prefix == null) {
+            data.scanner_prefix = "";
+        }
+        if (data.title == null) {
+            data.title = "";
+        }
+        if (data.description_observation == null) {
+            data.description_observation = "";
+        }
+        if (data.origin_component_name_version == null) {
+            data.origin_component_name_version = "";
+        }
+        if (data.origin_docker_image_name_tag == null) {
+            data.origin_docker_image_name_tag = "";
+        }
+        if (data.origin_endpoint_url == null) {
+            data.origin_endpoint_url = "";
+        }
+        if (data.origin_service_name == null) {
+            data.origin_service_name = "";
+        }
+        if (data.origin_source_file == null) {
+            data.origin_source_file = "";
+        }
+        if (data.origin_cloud_qualified_resource == null) {
+            data.origin_cloud_qualified_resource = "";
+        }
+        if (data.new_severity == null) {
+            data.new_severity = "";
+        }
+        if (data.new_status == null) {
+            data.new_status = "";
+        }
+        if (
+            !feature_vex_enabled() ||
```

Comment:
This condition doesn't work, the justification doesn't get saved. 

If you refactor `justificationEnabled` to a constant with the status as a parameter as suggested before, this line would work: 

`if (!justificationEnabled(data.new_status) || data.new_vex_justification == null) {`